js/map_utils.js

```diff
@@ -64,8 +64,10 @@ function addCentroids(map, countyCentroids) {
     .attr("r", function(d) { 
       return scaleCircles(d.properties[[activeCategory]]);
     })
+    .on("mouseover", function(d) { seeTooltip(this, d); })
+    .on("mouseout", function(d) { hideTooltip(this, d); })
```

Comment:
a question about these tooltips: they're currently attached to the circles, and i like what you're doing with bringing the hovered-over circle to the top. but how does and should this hover behavior interact with hovering over a county polygon? should/can we add the same hover functions to the polygons when those are ready? looking for your educated opinion here.

Reply:
🤔 thinking ....

Reply:
I think it could be easily done if I pull 

```
var orig = d3.select(currentCircle),
      origNode = orig.node();
  var duplicate = d3.select(origNode.parentNode.appendChild(origNode.cloneNode(true), 
                                                            origNode.nextSibling));
  
  // style circles
  orig
    .style("opacity", 0); // makes original circle invisible in the background
  duplicate
    .classed('county-point-duplicate', true)
    .style("pointer-events", "none")
    .style("opacity", 1); // makes the duplicate circle on the top
```
out to the `.on("mouseover"` specifically for the circle. Then we could do `.on("mouseover", function(d) { showTooltip(d); })` where ever the counties are added. The only thing that I don't think would work is that the `d` used here when it's associated with a circle has the actual data, but it wouldn't for the county boundary.

Reply:
ok, that sounds like a good approach. let's hold off until another PR regardless, but maybe what we need to do is index both the county topo and the centroid data on GEOID so we can quickly cross reference.

Reply:
what happens if there's a little circle that ends up getting completely swamped by a big circle? i imagine that as you move your mouse toward the little circle, it hits the big circle and brings that one to the front, such that it's actually impossible to get the little circle to show. do you think that's accurate? and does it happen at all / often enough to be a problem worth our attention?

Reply:
hmmm that's a good question. Doesn't appear to be a problem because I turned pointer events off for those duplicated circles, so there isn't any "hover" capability turned on when those are on top of everything else. Good thing to look into more though. Can we consider that with tooltip performance so that we can actually merge this and have people play around on dev?

Reply:
Here's the tiny dot I tested it on in california:
![image](https://user-images.githubusercontent.com/13220910/36869664-2cd4eaa0-1d62-11e8-917d-657d06c4e7ab.png)

Reply:
I could hover over it (mobile might be a different story though)

Reply:
cool, thanks for checking! yes, absolutely i'll merge soon, no refinement needed yet. just trying to figure out which refinements/planning will be needed.